### ISSUE TITLE
unseedable random state

### ISSUE THREAD

**Issue by vioSpark** [projectmesa/mesa#1138]
**Describe the bug**
space.py:459

**Expected behavior**
I should be able to give a seed for the random module

**To Reproduce**
call the find_empty() function multiple times

**Additional context**

**Comment by rht** [comment]
@vioSpark do you have a minimum viable sample code to reproduce the problem, and the error message?

**Comment by vioSpark** [comment]
@rht Tonight I tried to create a minimum viable sample code, but that passes all the tests. When I debugged what happened in our project's code base that could cause the error, I noticed that a teammate passed a tuple instead of the separate coordinates into mesa.grid.position_agent(), and that implicitly calls agent.random (space.py:512). The agent.random , which is a reference to agent.model.random (agent.py:29). While that random has its seed fixed in our code, for some reason it still behaves like it is unseeded. 

So to conclude, I couldn't build a reasonably small minimal viable sample under a considerable time, I'll probably try again some weeks later...
